CONTRIBUTING.md

```diff
@@ -246,6 +246,36 @@ This key identifies the section number from the portion of the ES6 standard that
 
 Read the [Test262 Technical Rationale Report](https://github.com/tc39/test262/wiki/Test262-Technical-Rationale-Report,-October-2017#specification-reference-ids) for reasoning behind deprecation.
 
+## Staging
+
+There is a `test/staging/` folder, containing tests that are subject to
+fewer requirements, in order to get tests running across implementations
+as early as possible, and promote interoperability of in-progress
+features.
+
+Tests in staging are not required to be split up into one test per file,
+or to conform to any particular style as long as they are runnable.
+In particular, mechanically-converted tests from implementations'
+private test suites are encouraged, as sharing them with other
+implementations promotes interoperability.
+
+These are the requirements for adding a test to staging:
+
+1. It tests a late-stage TC39 proposal or an existing language feature.
```

Comment:
Consider clarifying "late-stage"

Reply:
I realized "late-stage" is no different from the rule about what tests are [accepted](https://github.com/tc39/test262/blob/main/CONTRIBUTING.md#acceptable-tests) in the main tree, so I rephrased this.